script/bootstrap

```diff
@@ -8,11 +8,10 @@ else git clone https://github.com/PerfectFit-project/niceday-components.git buil
 fi
 
 
-cd build/niceday-components/niceday-broker/ || exit 1
+cd niceday-broker/ || exit 1
```

Comment:
Ah, this is what Candace also ran into I guess?

Reply:
And #13 fixes this in a nicer way?

Reply:
No, I think it's the other way around. This is showing up because of a merge conflict - Candace's fix is actually newer and works better. I'll fix this.